kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -129,6 +154,51 @@ private void setup(Map<String, Object> conf) {
     this.executor = MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workerThreadCount));
   }
 
+  /**
+   * Validate an individual cluster's configuration. A cluster's configuration is valid if:
+   * a) Looking it up by name in Burrow returns a successful response
+   * b) Bootstrap servers match those in burrow
+   *  i. in strict mode: The bootstrap server list in the config and the list returned from Burrow contain the same set
+   *  of servers
+   *  ii. in normal mode: All entries in the config's bootstrap server list are advertised by Burrow (Burrow could
+   * have additional entries which don't appear in the config).
+   *
+   * @param clusterConf the cluster configuration
+   * @return an optional string which will either contain the validation failure message if validation fails or be empty
```

Comment:
how about "a message describing the validation failure, if the config was invalid. Empty otherwise."

Reply:
fixed